src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -110,7 +110,11 @@ const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
           </li>
         )}
         renderInput={(params) => (
-          <TextField {...params} variant="standard" placeholder={`${reviewerIds.length} Reviewers Selected`} />
+          <TextField
+            {...params}
+            variant="standard"
+            placeholder={`${changeRequest.requestedReviewers.length} Reviewers Selected`}
```

Comment:
change this to reviewers.length so that the text updates in real-time when selecting and deselecting without having to submit cus the default value is already the changeRequests. test it out so u know what I mean.